{
  "vulinfo_id": "JVNDB-2018-012765",
  "title": "Perl におけるバッファエラーの脆弱性",
  "vulinfo_description": "Perl には、バッファエラーの脆弱性が存在します。",
  "affected": [
    {
      "name": "Canonical",
      "product_name": "Ubuntu",
      "cpe": {
        "version": "2.2",
        "text": "cpe:/o:canonical:ubuntu_linux"
      }
    },
    {
      "name": "Debian",
      "product_name": "Debian GNU/Linux",
      "cpe": {
        "version": "2.2",
        "text": "cpe:/o:debian:debian_linux"
      }
    },
    {
      "name": "The Perl Foundation",
      "product_name": "Perl",
      "version_number": [
        "5.26.3 未満",
        "5.28.1 未満の 5.28.x"
      ],
      "cpe": {
        "version": "2.2",
        "text": "cpe:/a:perl:perl"
      }
    }
  ],
  "impact": {
    "cvss": [
      {
        "version": "2.0",
        "severity": {
          "type": "Base",
          "text": "High"
        },
        "base": "7.5",
        "vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
      },
      {
        "version": "3.0",
        "severity": {
          "type": "Base",
          "text": "Critical"
        },
        "base": "9.8",
        "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
      }
    ],
    "impact_item": "情報を取得される、情報を改ざんされる、およびサービス運用妨害 (DoS) 状態にされる可能性があります。"
  },
  "solution": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。",
  "related": [
    {
      "type": "vendor",
      "name": "Debian Security Advisory",
      "vulinfo_id": "DSA-4347",
      "url": "https://www.debian.org/security/2018/dsa-4347"
    },
    {
      "type": "vendor",
      "name": "GitHub",
      "vulinfo_id": "Perl_my_setenv(); handle integer wrap",
      "url": "https://github.com/Perl/perl5/commit/34716e2a6ee2af96078d62b065b7785c001194be"
    },
    {
      "type": "vendor",
      "name": "The Perl Foundation",
      "vulinfo_id": "[SECURITY] Fedora 29 Update: perl-5.28.1-425.fc29",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/RWQGEB543QN7SSBRKYJM6PSOC3RLYGSM/"
    },
    {
      "type": "vendor",
      "name": "Ubuntu Security Notice",
      "vulinfo_id": "USN-3834-1",
      "url": "https://usn.ubuntu.com/3834-1/"
    },
    {
      "type": "vendor",
      "name": "Ubuntu Security Notice",
      "vulinfo_id": "USN-3834-2",
      "url": "https://usn.ubuntu.com/3834-2/"
    },
    {
      "type": "advisory",
      "name": "Common Vulnerabilities and Exposures (CVE)",
      "vulinfo_id": "CVE-2018-18311",
      "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-18311"
    },
    {
      "type": "advisory",
      "name": "National Vulnerability Database (NVD)",
      "vulinfo_id": "CVE-2018-18311",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2018-18311"
    },
    {
      "type": "advisory",
      "name": "関連文書",
      "vulinfo_id": "Bug 1646730",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1646730"
    },
    {
      "type": "cwe",
      "name": "JVNDB",
      "vulinfo_id": "CWE-119",
      "url": "https://jvndb.jvn.jp/ja/cwe/CWE-119.html",
      "title": "バッファエラー"
    }
  ],
  "history": [
    {
      "history_no": "1",
      "date_time": "2019-02-07T06:05:02Z",
      "description": "[2019年02月07日]\\n  掲載"
    }
  ],
  "date_first_published": "2019-02-07T15:05:02+09:00",
  "date_last_updated": "2019-02-07T15:05:02+09:00",
  "date_public": "2018-12-02T00:00:00+09:00"
}
